### ISSUE TITLE
LD_LIBRARY_PATH broken

### ISSUE THREAD

**Issue by nonchip** [lutris/lutris#781]
tl;dr: the manual env setting for `LD_LIBRARY_PATH` (because only for "native linux games" there's a dedicated setting for appending stuff to that variable) has no effect at all.

if I set LD_LIBRARY_PATH in the environment variables list manually to try and include `/usr/lib32` to see if it fixes retroarch (which is still utterly broken btw), and then echo it from the "run in terminal"-shell, because it didn't help, I see, LITERALLY:
```
$ echo $LD_LIBRARY_PATH
/home/kyra/.local/share/lutris/runtime/lib32:/home/kyra/.local/share/lutris/runtime/steam/i386/lib/i386-linux-gnu:/home/kyra/.local/share/lutris/runtime/steam/i386/lib:/home/kyra/.local/share/lutris/runtime/steam/i386/usr/lib/i386-linux-gnu:/home/kyra/.local/share/lutris/runtime/steam/i386/usr/lib:/home/kyra/.local/share/lutris/runtime/lib64:/home/kyra/.local/share/lutris/runtime/steam/amd64/lib/x86_64-linux-gnu:/home/kyra/.local/share/lutris/runtime/steam/amd64/lib:/home/kyra/.local/share/lutris/runtime/steam/amd64/usr/lib/x86_64-linux-gnu:/home/kyra/.local/share/lutris/runtime/steam/amd64/usr/lib:$LD_LIBRARY_PATH
```

yeah obviously that can't work, because someone seems to have thought python supports shell expansion in https://github.com/lutris/lutris/blob/master/lutris/runtime.py#L135, which is obviously not the case.

probably replacing that `$LD_LIBRARY_PATH` literal with something like `os.getenv('LD_LIBRARY_PATH', '')` would fix this.

EDIT: tried that replacement locally, doesn't help, apparently the environment variable from the config isn't actually set at that point, so it would have to be added to the dicts in `get_paths` somehow.

EDIT: without the runtime, the env variable **isn't set at all**, so apparently somewhere it gets overwritten unconditionally.

**Comment by strycore** [comment]
I thought I had another change to make (in thread.py, we manipulate LD_LIBRARY_PATH as well) but the code there is already good.
